4-esqueleto/README.md

```diff
@@ -1,3 +1,274 @@
+
+La agenda es una sección dentro de la web, es decir, tiene su propia página, por lo que si un usuario hace clic en el título “Agenda”, se le enviará a la página en la que se encuentran todas las entradas relacionadas con esta sección. Primero aparecerán los cinco primeros acontecimientos que aparecen en la página principal seguidos de futuros acontecimientos. La forma en la que se mostrará el contenido será en Scroll infinito; esto significa que  el usuario verá todos los acontecimientos a medida que vaya descendiendo, en orden cronológico de lo más reciente a lo más próximo. Los acontecimientos que ya se hayan celebrado se eliminarán automáticamente de la lista.
+
+Todos los usuarios, independientemente de si están registrados o no, podrán acceder a los contenidos que se muestran en la agenda. 
+
+![Agenda003](https://github.com/DeustoPWEB2018/proyectoweb-ocio/blob/anejuaristi-esqueleto/4-esqueleto/imagenes/Agenda003.jpg)
+
+
+### Videoclips
+
+En la sección “Videoclips”, el usuario encontrará una selección de vídeos musicales que nosotros habremos escogido previamente. Nuestros criterios a la hora de seleccionarlos no solo responden a la popularidad de los vídeos o a la de los artistas que lo interpretan, sino que también valoramos su originalidad y calidad artística, así como aquello que creamos interesante para nuestros visitantes, ya que son personas con un perfil curioso, pero también exigente. 
+
+Los vídeos se cogerán de la plataforma Youtube, de modo que nosotros tan solo tendremos que insertar el enlace correspondiente dentro de nuestra web para que, una vez que el usuario haga “play” sobre él, éste se reproduzca directamente, por lo que el usuario no tiene que entrar en Youtube, aunque en caso de quiera hacerlo, tan solo tendrá que pulsar el botón derecho de su ratón y hacer clic en la opción “abrir enlace en una nueva pestaña”. 
+
+Los vídeos se mostrarán siguiendo una jerarquía que responde a un orden cronológico desde lo más reciente, a lo más antiguo. La idea, es que cada día se añada un videoclip nuevo por sección, a modo de que la sección esté constantemente ofreciendo nuevo contenido e incite al usuario a navegar por la misma. 
+
+Respecto a lo que el usuario observará, éste verá el título del vídeo, la caja con el vídeo y, debajo de éste, un pequeño párrafo con información más detallada sobre él; podría ser una presentación del vídeo con una anécdota, por ejemplo. Al final de cada párrafo, el usuario observará las etiquetas que se han utilizado para categorizar el vídeo dentro de la página, a modo de relacionarlo con contenido similar. También habrá iconos sociales para que el usuario, no sólo comparta el vídeo, sino que también comparta nuestras aportaciones. 
+
+![videoclips](https://github.com/DeustoPWEB2018/proyectoweb-ocio/blob/anejuaristi-esqueleto/4-esqueleto/imagenes/Ejemplo%20contenido001.jpg)
+
+## Diseño de la información
```

Comment:
El diseño de la información no se refiere solamente a las etiquetas que aportáis al contenido (a modo de sistema de clasificación abierto), sino también a los términos que utilizáis para refereiros a los diferentes elementos del sitio web. Por ejemplo:

- "Skip ad" en vez de "Saltar anuncio", es diseño de la información.
- El icono de la lupa (¿cuál en concreto?) y el icono de la "personita", son diseño de la información.
- _Actualidad_ en vez de _Noticias_; es diseño de la información.
- _Agenda_ en vez de _Eventos_;
- _Acceso_ en vez de _Entrar_;
- _Conócenos_ en vez de _Acerca de nosotros_;
- ...

Todo esto es diseño de la información.